src/Sylius/Behat/Page/Admin/Product/FormTrait.php

```diff
@@ -13,8 +13,11 @@
 
 namespace Sylius\Behat\Page\Admin\Product;
 
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
 use Sylius\Behat\Service\DriverHelper;
 use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
 
 trait FormTrait
```

Comment:
I would be for replacing a FormTrait by a FormElement 🖖🏻

Reply:
I'm planning it for our global unification.